src/components/PokeList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+// import { usePokeContex } from '../context/PokeContext';
+// import './Component.css';
+
+export default function PokeList({grab}) {
```

Comment:
Naming conventions are super important. Be sure to add in relevant naming conventions for other engineers who may be handling your work in the future.